src/primevue/toast/toast.ts

```diff
@@ -0,0 +1,62 @@
+import { ToastPassThroughOptions } from "primevue/toast";
+import { tw } from "@/lib/tags.ts";
+
+const toast: ToastPassThroughOptions = {
+    message: ({ props }) => {
+        const base = tw`flex flex-row items-center gap-8 border-l-8 p-16`;
+
+        const severity = props.message?.severity;
+
+        const success = tw`border-l-green-900 bg-green-200`;
+        const error = tw`border-l-red-900 bg-red-200 text-red-900`;
+        const warn = tw`border-l-yellow-900 bg-yellow-200 text-yellow-900`;
+
+        return {
+            class: {
+                [base]: true,
+                [success]: severity === "success",
+                [error]: severity === "error",
+                [warn]: severity === "warn",
+            },
+        };
+    },
+
+    messageContent: () => ({
+        class: tw`flex flex-row items-start gap-8`,
+    }),
+
+    messageIcon: ({ props }) => {
+        const baseIcon = tw`p-2 w-20 h-20`;
+
+        const severity = props.message?.severity ?? "info";
+
+        const successIcon = tw`text-green-900`;
+        const errorIcon = tw`text-red-800`;
+        const warnIcon = tw`text-yellow-800`;
+        const infoIcon = tw`text-blue-800`;
+
+        return {
+            class: {
+                [baseIcon]: true,
+                [successIcon]: severity === "success",
+                [errorIcon]: severity === "error",
+                [warnIcon]: severity === "warn",
+                [infoIcon]: severity === "info",
+            },
+        };
+    },
+
+    messageText: () => ({
+        class: tw`text-black`,
+    }),
+
+    summary: () => ({
+        class: tw`font-bold`,
```

Comment:
Todo: please use the typography styles. You can see which one in Figma, in RIS UI they're defined in the Tailwind plugin. Please also check the other texts and use the correct classes, even if it already looks right by default. This is so that when we need to change typography styles we only need to update them in one central location.